src/foam/u2/view/UnstyledTableView.js

```diff
@@ -449,7 +449,7 @@ foam.CLASS({
                   }).
                   callIf(view.dblclick && ! view.disableUserSelection, function() {
                     tableRowElement.on('dblclick', function() {
-                      view.dblclick && view.dblclick(null, obj.id);
+                      view.dblclick && view.dblclick(view.dataChangedFlag ? null : obj, obj.id);
```

Comment:
why do we need to change code here?

we don't have a whole object in table we have only properties set, which are table columns, that's way we were passing null